templates/Content/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/_BenchmarkProjectName_.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Currently, `dotnet build` on the generated template produces output in `bin\Debug` which may be confusing because we have optimized assemblies. It will be better to include `<Configuration>Release</Configuration>` in all csproj to avoid such kind of confusions.

Reply:
Makes sense - the PR update will follow in few seconds...